docs/articles/faq/README.md

```diff
@@ -283,3 +283,17 @@ According to users' feedback, the following CI systems work best with TestCafe:
 * [TravisCI](../documentation/guides/continuous-integration/travis.md)
 * [CircleCI](../documentation/guides/continuous-integration/circleci.md)
 * [AppVeyor](../documentation/guides/continuous-integration/appveyor.md)
+
+### 'The browser can't open the page: can't establish a secure connection to the server'
+
+If your computer is connected to multiple networks (for instance, uses a VPN connection), TestCafe may host its internal proxy server on an IP that cannot be routed correctly due to VPN. In this case, the following error occurs:
+
+```text
+<browsername> can't open the page because <browsername> can't establish a secure connection to the server.
+```
+
+To explicitly state that the proxy should be hosted on the local machine, launch TestCafe with the `--hostname localhost` CLI option:
```

Comment:
`To explicitly state that the proxy should be hosted on the local machine`  - it's not correct.
The 'hostname' parameter just guarantees that the proxied URL starts with `http://localhost/site-utl` instead of `http://192.168.8.8/site-ur` ones.

I propose to use the common phrase: `To fix the problem, launch TestCafe with...`

Reply:
> To use `localhost` in TestCafe URLs, ...

?

Reply:
For which purposes I should `To use localhost in TestCafe URLs, ..`?

My variant tells why you should use it.

Reply:
You have a point

Reply:
```suggestion
To fix the issue, launch TestCafe with the `--hostname localhost` CLI option:
```